README.md

```diff
@@ -40,6 +40,7 @@ terraform {
   required_providers {
     astro = {
       source = "astronomer/astro"
+      version >= "0.0.0"
```

Comment:
you don't need a version
without version, it gets the latest one already

Reply:
Iv had customers cpmplain that this is not the case

Reply:
are they using the one on the registry?
i just tested it and it worked fine without version on a brand new terraform project

Reply:
maybe they were doing something else wrong. I haven't heard this complaint in awhile